### PR TITLE
docs: new release info on the F36 website

### PR DESCRIPTION
**Pull request by burakukula**
Hey there!

Just a small update for the website to display information about the new release for F36

Screenshot 2021-01-21 at 14 13 34

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

**Review by domarku — APPROVED** [review]
🚀

File: packages/forma-36-website/gatsby-config.js [hunk 1]
```diff
@@ -3,9 +3,11 @@ const path = require('path');
 module.exports = {
   siteMetadata: {
     title: 'Forma 36 - The Contentful Design System',
-    promoText: '',
-    promoLink: '',
-    promoLinkText: '',
+    promoText: 'Introducing a visual refresh to Forma 36 components',
```
Comment (denkristoffer) [inline packages/forma-36-website/gatsby-config.js:6]: Should we call them React components like we do elsewhere?

Comment (burakukula) [reply packages/forma-36-website/gatsby-config.js:6]: hmm, good point. On the website, the audience is not only engineers, so maybe it's not so important information, but sure, for consistency reasons I can adjust it.

File: packages/forma-36-website/src/components/Promo.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import tokens from '@contentful/forma-36-tokens';
+import { Tag } from '@contentful/forma-36-react-components';
+import { Flex } from '@contentful/forma-36-react-components/dist/alpha';
```
Comment (denkristoffer) [inline packages/forma-36-website/src/components/Promo.js:4]: We don't need the alpha anymore btw 🥳

Comment (denkristoffer) [reply packages/forma-36-website/src/components/Promo.js:4]: Or do we? I see you added it elsewhere.

Comment (burakukula) [reply packages/forma-36-website/src/components/Promo.js:4]: So... this is not working then. This is throwing me error when I try to build website.

Comment (burakukula) [reply packages/forma-36-website/src/components/Promo.js:4]: maybe something is wrong on in the website, I will have another look. thx! Forgot that we already have it in the main build.

Comment (burakukula) [reply packages/forma-36-website/src/components/Promo.js:4]: AAAA ok, 🤦‍♀️  I know what's the issue. @denkristoffer thx!

Comment (burakukula) [reply packages/forma-36-website/src/components/Promo.js]: ✅

File: packages/forma-36-website/gatsby-config.js (hunk 1 above)
Comment (burakukula) [reply packages/forma-36-website/gatsby-config.js]: @denkristoffer ✅

**Review by denkristoffer — APPROVED** [review]
🎉 🙏 🙇

**Pull request merged**